### DISCUSSION TITLE
Stepper width

### DISCUSSION THREAD

**Discussion by gittrolo** [quasarframework/quasar#9098] [General - Components / Directives / etc]
I want the stepper header to always be on a single line. In the attached example step number 4 goes on a new line, I want it to be in the same line as the other steps. How can I reduce the width of the lines separating the steps? 
![image](https://user-images.githubusercontent.com/52664193/116551920-b8c1da80-a8f8-11eb-810b-2ac0e9555be3.png)

**Comment by metalsadman** [comment]
inspect element, and override css. better put that in a codepen, so peeps can play around with it if you can't do it on your own.

**Reply by gittrolo** [reply]
i tried to inspect the code, but i'm not very good at it. 
you can see the code by going https://quasar.dev/vue-components/stepper#example--alternative-labels
and clicking on the button "Edit in codepen"
